### PR TITLE
Makes the drawing tablet not GARBAGE

### PR DESCRIPTION
**Pull request by GuppyLaxx**
I was observing poor Vex and then realized the lad had a drawing tablet, and it interested me! And then I looked into it and NOTICED IT WAS A GOD DAMN GENERIC ITEM. SO I FIXED IT.
PiJuvZkBgS

:cl: GuppyLaxx
add: Added actual use for Vex's drawing tablet!
/:cl:

### CODE REVIEW

**Comment by GuppyLaxx** [comment]
bro why is THIS RANDOM ASS LINTER FAILING ME

File: modular_skyrat/modules/customization/modules/client/loadout/donator/personal/donator_personal.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /datum/loadout_item/backpack/donator/drawingtablet
 	name = "Drawing Tablet"
-	path = /obj/item/drawingtablet
+	path = /obj/item/canvas/drawingtablet/
```
Comment (KasparoVy) [inline modular_skyrat/modules/customization/modules/client/loadout/donator/personal/donator_personal.dm]: *shrug* linter might not like the `/` at the very end of this path.

**Comment by KasparoVy** [comment]
Awesome job, good call using canvascode

**Comment by GuppyLaxx** [comment]
Also pls don't merge for now, I want to try and update its sprite too

**Comment by NotRanged** [comment]
image

**Comment by GuppyLaxx** [comment]
Sadly, I won't be getting the sprite any time soon. Feel free to merge this if it doesn't piss off the linter

**Comment by Useroth** [comment]
Pretty good. Ding dong mergu.

**Pull request merged**